packages/odyssey-react/jest.d.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare const a11yCheck: (renderFn: () => unknown) => void
```

Comment:
I would have prefered to annotate the return value of the callback here to `RenderResult` but was unable to get the triple slash directive working as I wanted. The compiler could not find that exported type.

```
/// <reference types="@testing-library/react" />
```

Reply:
Thanks @conorhanrahan-okta. My understanding is that by using an `import` (or `export` for that matter) the compiler will evaluate any given file in Module Mode. This is typically how I write TS code, but it also prevents ambient declarations from taking effect! To have an ambient variable, type, or module declared it has to be placed within a file that is evaluated in Script Mode, meaning no `import` or `export` statements. This is why I reached for the triple slash directive so that I can still compose 3rd party types but do so in a way that doesn't prevent the ambient variable declaration from taking effect. At least that was the plan 😅

Reply:
Ah interesting, I didn't know about that import/export in Script Mode stuff. That explains some of the other weird type errors in other files I was getting. Tricky...